app/models/order.rb

```diff
@@ -34,4 +34,11 @@ def cancel_items
     end
   end
 
+  def current_status
+    if self.status == "Pending"
+      self.update(status: 0) if item_orders.select(:status).distinct.pluck(:status) == ["Fulfilled"]
+    end
+    self.status
```

Comment:
We had two versions of current status and are going to move forward with this method.